examples/benchmark.py

```diff
@@ -127,7 +127,6 @@
 
         for params in params_sweep:
             _, _, _, res = run(**params)
-            clear_cache()
```

Comment:
Are you completely sure about this? The problem with memory leaks was mainly that SymPy was keeping an additional reference to our objects in it's internal cache, tricking the Python GC to never collect our objects, and thus never calling `__del__` on them. I don't think it had anything to do with us not cleaning the C memory, since the previous implementation was based on natively created NumPy arrays, so NumPy would free the C pointer once the GC was invoked on our symbols and their NumPy arrays.